beacon-chain/blockchain/service.go

```diff
@@ -71,7 +71,8 @@ type config struct {
 	ChainStartFetcher       execution.ChainStartFetcher
 	BeaconDB                db.HeadAccessDatabase
 	DepositCache            cache.DepositCache
-	ProposerSlotIndexCache  *cache.ProposerPayloadIDsCache
+	PayloadIDCache          *cache.PayloadIDCache
+	TrackedValidatorsCache  *cache.TrackedValidatorsCache
```

Comment:
I like the trackedValidators name because we can use this for everything in the beacon. I plan to move to use this also by default in the monitor for example.

Reply:
makes sense if we are going to extend this to attester and other duties as well!